packages/actions/src/releasePackages/index.ts

```diff
@@ -0,0 +1,18 @@
+import process from 'node:process';
+import { getInput } from '@actions/core';
+import { generateReleaseTree } from './generateReleaseTree.js';
+import { releasePackage } from './releasePackage.js';
+
+const argInput = process.argv[2];
+const useableArg = argInput === 'discord.js' || argInput?.startsWith('@discordjs/');
+const packageName = getInput('package') || useableArg ? argInput : '';
```

Comment:
Is it actually possible that `getInput('package')` will be falsy? Also, should `useableArg` account for create-discord-bot?

Reply:
yes, getInput returns an empty string when no input is passed (so when run from the command line and when you don't select a specific package in the workflow dispatch)